charts/matrix-stack/templates/postgres/_helpers.tpl

```diff
@@ -114,7 +114,7 @@ true
                         "LC_COLLATE" "C"
                         "LC_CTYPE" "C"
 -}}
-{{- $resultEnv := merge $resultEnv $overrideEnv -}}
+{{- $resultEnv := mustMergeOverwrite $overrideEnv $resultEnv -}}
```

Comment:
I don't think this is particularly worth having a test for. Agreed?

Reply:
Well actually it was buggy : 
 - We did not have `extraEnv` on postgres
 - The override was wrongly ordered